config/schemas/config.yaml

```
@@ -1962,6 +1962,10 @@ properties:
           Sync object storage from the primary site to a secondary site with Rclone.
         type: object
         properties:
+          activeDeadlineSeconds:
+            title: Rclone cronjob activeDeadlineSeconds
```

Schema LGTM. Possible title improvement:

```suggestion
            title: Rclone Cronjob Active Deadline Seconds
```

---

If I had no idea what active deadline seconds was this wouldn't help much. What about something like `The maximum amount of time that the Rclone job is allowed to run (in seconds).`?

---

Ah sorry, this was the title, not the description. Ignore my previous comment. :smile:

---

A `description` with a link to <https://kubernetes.io/docs/concepts/workloads/controllers/job/> perhaps?